docs/cep/query-guide/functions/map/putIfAbsent.md

```diff
@@ -21,8 +21,27 @@ Function returns the updated map after adding the given key-value pair if key is
 ## Example 1
 
 ```sql
-@info(name = 'query1')
 map:putIfAbsent(stockDetails , 1234 , 'IBM')
 ```
 
-The `map:putIfAbsent(stockDetails, 1234, 'IBM')` function takes a map named `stockDetails`, a key `1234`, and a value `'IBM'`. If the key `1234` is not already present in the `stockDetails` map, then it adds the key-value pair (1234, 'IBM') to the map. If the key `1234` is already present in the map, then the function will not change the map. The function returns the updated `stockDetails` map.
+The `map:putIfAbsent(stockDetails, 1234, 'IBM')` function checks the `stockDetails` map for the presence of the key `1234`. If the key `1234` is not present in the map, it inserts the key-value pair `1234`:`'IBM'` into the map. However, if the key `1234` already exists in the map, the map remains unchanged. The function returns the `stockDetails` map after the operation.
+
+## Example 2
+
+```sql
```

Comment:
This one isn't working either.

Reply:
@diana-macrometa you used stockSymbol in UpdatedStockData definition but SELECTED and inserted simply "symbol" , you can change in the definition of UpdatedStockData "stockSymbol" to simply symbol